src/gesture-recognizers.js

```diff
@@ -271,8 +273,10 @@ class SwipeRecognizer extends GestureRecognizer {
 
   /** @override */
   onTouchMove(e) {
-    const touches = e.changedTouches || e.touches;
```

Comment:
So `changedTouches` refers to all touches involved in this event. Aka touches that moved, initiated, or were removed. `touches` refers to all contact points with the screen. This changes the functionality of swipe so that it only works when one finger is on the screen--basically no matter how many `changedTouches` there are, if more than one touch is on the screen, then we don't register a swipe. This was done because a large number of pinches were aggressively misread as swipes. 

For `TapZoom`, I'm not sure if we consider one finger on the screen (not moving), and a second finger moving a `TapZoom` or a `Pinch`. If we assign that to `Pinch` and not `TapZoom`, then we can make the same change to `TapZoom`.

Reply:
Cool, that makes sense. My initial impression is that one stationary finger and one moving finger should be a pinch zoom and not a tap zoom. macOS seems to require both fingers moving, but iOS's default behavior seems to allow one finger stationary or both fingers moving to pinch zoom

Reply:
That makes sense. I've gone ahead and updated `TapZoom` to `const touches = e.touches` as well.